guides/guide-for-adding-windows-node.md

```diff
@@ -65,3 +65,4 @@ This guide offers two choices:
 
```

Comment:
nit about the line above: should become "This guide offers **three** choices:"

Reply:
Thanks, updated it!